docs/standard/serialization/snippets/system-text-json-how-to-5-0/csharp/QuotedNumbers.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace QuotedNumbers
+{
+    public class Forecast
+    {
+        public DateTime Date { get; init; }
+        public int TemperatureC { get; set; }
+        public string Summary { get; set; }
+    };
+
+    public class Program
+    {
+        public static void Main()
+        {
+            Forecast forecast = new()
+            {
+                Date = DateTime.Now,
+                TemperatureC = 40,
+                Summary = "Hot"
+            };
+
+            JsonSerializerOptions options = new()
+            {
+                NumberHandling = JsonNumberHandling.WriteAsString,
+                WriteIndented = true
+            };
+
+            string forecastJson = JsonSerializer.Serialize<Forecast>
+                (forecast, options);
+            Console.WriteLine($"Output JSON:\n{forecastJson}");
+
+            options = new JsonSerializerOptions()
+            {
+                NumberHandling = JsonNumberHandling.AllowReadingFromString
+            };
```

Comment:
Can we use the same `options` instance, but set `NumberHandling` to `JsonNumberHandling.AllowReadingFromString | JsonNumberHandling.WriteAsString`?

Reply:
That results in `System.InvalidOperationException: Serializer options cannot be changed once serialization or deserialization has occurred.`

Reply:
Thanks!